README.md

```diff
@@ -59,6 +59,7 @@ A curated list of awesome robot descriptions in URDF, Xacro or MJCF formats.
 | Name | Maker | Formats | License | Visuals | Inertias | Collisions |
 |------|-------|---------|---------|---------|----------|------------|
 | Bolt | ODRI | [URDF](https://github.com/Gepetto/example-robot-data/tree/master/robots/bolt_description) | BSD-3-Clause | ✔️ | ✔️ | ✔️ |
+| BD-X | Disney | [URDF](https://github.com/rimim/AWD/tree/main/awd/data/assets/go_bdx) [MJCF](https://github.com/rimim/AWD/tree/main/awd/data/assets/go_bdx) | NVIDIA License | ✔️ | ✔️ | ✔️ |
```

Comment:
```suggestion
| BD-X | Disney | [URDF](https://github.com/rimim/AWD/blob/main/awd/data/assets/go_bdx/go_bdx.urdf) [MJCF](https://github.com/rimim/AWD/blob/main/awd/data/assets/go_bdx/go_bdx.xml) | NVIDIA License | ✔️ | ✔️ | ✔️ |
```

Repeated links won't pass awesome-lint checks. One trick is to use more specific links like so.